scripts/analyze_training_metrics.py

```diff
@@ -124,7 +124,16 @@ def analyze_metrics(
 
         date, model_name, metric = parse_metric_file(metric_file_path)
 
+        is_binary = len(metric["report"]["targets"]) == 2
         # Then process the report
+        if is_binary:
+            for target in range(2):
```

Comment:
Instead of "hardcoding" the class names like this, you could do `for target in metric["report"]["targets"]:` or even better `for target, target_metrics in metric["report"]["targets"].items():`, so in the next line you can do `for key, value in target_metrics.items():` instead of `for key, value in metric["report"]["targets"][str(target)].items():`.

Reply:
Thank you for the tip!